blockcypher/api.py

```diff
@@ -1499,6 +1500,15 @@ def create_unsigned_tx(inputs, outputs, change_address=None,
             inputs_cleaned.append({
                 'addresses': [address, ],
                 })
+        elif 'prev_hash' and  'output_index' in input_obj:
+            prev_hash = input_obj['prev_hash']
+            output_index = input_obj['output_index']
+            assert uses_only_hash_chars(prev_hash), prev_hash
+            assert str(output_index).isdigit(), output_index
```

Comment:
Awesome regexes, thanks for the tip.

Will merge once test is in. Thanks again.